<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <!-- Arguments -->
  <arg name="launch_rviz" default="true"/>
  <arg name="cloud_topic" default="/colorized_point_cloud"/>
  <arg name="parameter_filename" default="param_velodyne_puck16_denseMap.lua"/>
  <arg name="parameter_folder_path" default="$(find open3d_slam_ros)/param/"/>
  <arg name="is_read_from_rosbag" default="false"/>
  <arg name="rosbag_filepath" default=""/>
  <arg name="use_sim_time" default="true"/>

  <!-- Launch ROS Node and pass all arguments -->
  <include file="$(find open3d_slam_ros)/launch/examples/anymal.launch" pass_all_args="true"/>

</launch>